Sources/Network/Connection/Connections/ConsensusConnection.swift

```diff
@@ -15,23 +15,17 @@ final class ConsensusConnection: Connection<
     private let grpcFactory: GrpcProtocolConnectionFactory
     private let config: NetworkConfig
     private let targetQueue: DispatchQueue?
-    private let rng: (@convention(c) (UnsafeMutableRawPointer?) -> UInt64)?
-    private let rngContext: Any?
```

Comment:
Why do we need to remove these ?

Reply:
Look at all XXXConnection classes

Reply:
in PR #188, I restored all the rng code that was in place prior to starting idempotence.